roles/maas_firewall/tasks/teardown.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Remove iptables rules as part of teardown
+
+- name: Set policy for INPUT chain to accept
+  ansible.builtin.iptables:
+    chain: INPUT
+    policy: ACCEPT
+
+- name: Set policy for FORWARD chain to drop
+  ansible.builtin.iptables:
+    chain: FORWARD
+    policy: ACCEPT
```

Comment:
This is doing accept instead of drop, which is the default rule for FORWARD, I think you just want to rename the task here.

Reply:
Thanks, I copied across and that slipped through